packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -10,7 +15,7 @@ type Props = {
     /**
      * The form field component.
      */
-    field: React.ReactNode;
+    field: React.ReactElement;
```

Comment:
Needed to update the `field` prop to be the `ReactElement` type in order to use `React.cloneElement` to apply attributes from this component. I think this makes sense for the `field` prop so that it can't be `boolean | null | undefined` which `ReactNode` allows.

Let me know if there are any concerns with this!